### PR TITLE
docs: fix multiple typos

### PR DESCRIPTION
**Pull request by osrm**
# Summery

Fix typos

# Description

I've checked every ‘md’ files in the repository, found some typos that needed to be fixed because they clearly changed the meaning, and fixed them.

There were also some awkward wording and punctuation, but I didn't fix those because they don't detract too much from the meaning of the original documentation and don't cause any problems in understanding the documentation.

In the commit, I briefly mentioned the typo corrections and the reasons why they were typos. 

Hope this helps.

### CODE REVIEW

File: docs/nbc_audit_2020/eth2_spec_core/reward_processing.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Writeup:
 *However, around 11 AM UTC on May 15th , it had been observed that Schlesi had
 been nearly 50 epochs away from finality. While the initial cause of the finality delay has not been ascertained, they’re not rare in a test
 network where participants are not obligated/incentivized to keep their
-validators running. The lack of finality eventually lead to instability
+validators running. The lack of finality eventually led to instability
```
Comment (tersec) [inline docs/nbc_audit_2020/eth2_spec_core/reward_processing.md]: `docs/nbc_audit_2020` is of legacy/historical interest only and is not being modified anymore.

Comment (osrm) [reply docs/nbc_audit_2020/eth2_spec_core/reward_processing.md:25]: I understand, Would it be acceptable to open the PR excluding that commit?

Comment (osrm) [reply docs/nbc_audit_2020/eth2_spec_core/reward_processing.md]: Should I cancel the proposed changes, then?

Comment (tersec) [reply docs/nbc_audit_2020/eth2_spec_core/reward_processing.md]: that would probably be for the best, yes.

Comment (osrm) [reply docs/nbc_audit_2020/eth2_spec_core/reward_processing.md]: Reverted back to the way it was before it was fixed.

File: docs/the_auditors_handbook/src/03.5_the_threat_model.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This includes but not limited to:
 
 In particular, we do not defend against a malicious cloud provider which would offer
 a “Rent-a-Raspberry-Pi” service for validators and would subsequently engage in attacks against
-those validators, the implest being, taking the validator offline when it's their duty time.
```
Comment (tersec) [inline docs/the_auditors_handbook/src/03.5_the_threat_model.md:21]: The auditors handbook is archived and not maintained anymore.

File: docs/the_auditors_handbook/src/03_nbc_nimbus_beacon_chain.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Nimbus Beacon Chain
 
 !!! warning
-    This auditors' handbook is frozen and obsolete; the [Nim language manual](https://nim-lang.org/docs/manual.html) alongside [other Nim documentation](https://nim-lang.org/documentation.html), [Status Nim style guide](https://status-im.github.io/nim-style-guide/), [Chronos guides](https://github.com/status-im/nim-chronos/blob/master/docs/src/SUMMARY.md), and [Nim by Example](https://nim-by-example.github.io/getting_started/) supercede it.
+    This auditors' handbook is frozen and obsolete; the [Nim language manual](https://nim-lang.org/docs/manual.html) alongside [other Nim documentation](https://nim-lang.org/documentation.html), [Status Nim style guide](https://status-im.github.io/nim-style-guide/), [Chronos guides](https://github.com/status-im/nim-chronos/blob/master/docs/src/SUMMARY.md), and [Nim by Example](https://nim-by-example.github.io/getting_started/) supersede it.
```
Comment (tersec) [inline docs/the_auditors_handbook/src/03_nbc_nimbus_beacon_chain.md:4]: The auditors handbook is archived and not maintained anymore.

**Comment by tersec** [comment]
Thank you for the documentation typo fix; the one which is not in a frozen part of the repo (`advanced`) has been updated in `unstable`.

**Pull request closed without merging**